python_tests/test_torchscript.py

```diff
@@ -873,7 +873,9 @@ def test_unary_ops(self):
         GRAPH_EXECUTOR != ProfilingMode.PROFILING,
         "Requires fusion optimization pass to be effective",
     )
-    def test_category_rule(self):
+    # Disabled due to change in torch.testing._internal.jit_metaprogramming_utils
+    # See https://github.com/NVIDIA/Fuser/issues/624
+    def _test_category_rule(self):
```

Comment:
nitpick, we can probably a proper decorator other than hacking the test name.
`@unittest.skipIf(True, "disabling TorchScript failure See https://github.com/NVIDIA/Fuser/issues/624")`

Reply:
Oh of course. That's a good call. Will update and merge after dinner.

Reply:
```suggestion
    def test_category_rule(self):
```

Reply:
With `@skipIf`, we can name these back to `test_*`

Reply:
Oops, somehow I missed that. I'll double check those when CI finished. Thanks for catching this one.